lib/modules/platform/github/api-cache.ts

```diff
@@ -0,0 +1,84 @@
+import { DateTime } from 'luxon';
+import type { ApiPageCache, ApiPageItem } from './types';
+
+export class ApiCache<T extends ApiPageItem> {
+  constructor(private cache: ApiPageCache<T>) {}
+
+  getEtag(): string | null {
+    return this.cache.etag ?? null;
+  }
+
+  setEtag(value: string | null): void {
+    if (value === null) {
+      delete this.cache.etag;
+    } else {
+      this.cache.etag = value;
+    }
+  }
```

Comment:
```suggestion
  get etag(): string | null {
    return this.cache.etag ?? null;
  }

  set etag(value: string | null) {
    if (value === null) {
      delete this.cache.etag;
    } else {
      this.cache.etag = value;
    }
  }
```

I mean real getter and setter 😉

Reply:
Haha